fpga/sim/vcompile/ips.mk

```diff
@@ -148,23 +149,25 @@ clean:
 	@make --no-print-directory -f $(mkfile_path)/ips/udma_camera.mk clean
 	@make --no-print-directory -f $(mkfile_path)/ips/udma_filter.mk clean
 	@make --no-print-directory -f $(mkfile_path)/ips/udma_external_per.mk clean
+	@make --no-print-directory -f $(mkfile_path)/ips/udma_hyper.mk clean
 	@make --no-print-directory -f $(mkfile_path)/ips/hwpe-ctrl.mk clean
 	@make --no-print-directory -f $(mkfile_path)/ips/hwpe-stream.mk clean
 	@make --no-print-directory -f $(mkfile_path)/ips/hwpe-mac-engine.mk clean
 	@make --no-print-directory -f $(mkfile_path)/ips/riscv-dbg.mk clean
+	@make --no-print-directory -f $(mkfile_path)/ips/register_interface.mk clean
 	@make --no-print-directory -f $(mkfile_path)/ips/pulp_soc.mk clean
 	@make --no-print-directory -f $(mkfile_path)/ips/axi2mem.mk clean
 	@make --no-print-directory -f $(mkfile_path)/ips/axi2per.mk clean
 	@make --no-print-directory -f $(mkfile_path)/ips/per2axi.mk clean
-	@make --no-print-directory -f $(mkfile_path)/ips/axi_size_conv.mk clean
-	@make --no-print-directory -f $(mkfile_path)/ips/cluster_interconnect.mk clean
 	@make --no-print-directory -f $(mkfile_path)/ips/event_unit_flex.mk clean
 	@make --no-print-directory -f $(mkfile_path)/ips/mchan.mk clean
 	@make --no-print-directory -f $(mkfile_path)/ips/hier-icache.mk clean
 	@make --no-print-directory -f $(mkfile_path)/ips/icache-intc.mk clean
 	@make --no-print-directory -f $(mkfile_path)/ips/icache_mp_128_pf.mk clean
 	@make --no-print-directory -f $(mkfile_path)/ips/icache_private.mk clean
+	@make --no-print-directory -f $(mkfile_path)/ips/hci.mk clean
 	@make --no-print-directory -f $(mkfile_path)/ips/cluster_peripherals.mk clean
 	@make --no-print-directory -f $(mkfile_path)/ips/fpu_interco.mk clean
+	@make --no-print-directory -f $(mkfile_path)/ips/hwpe-datamover-example.mk clean
 	@make --no-print-directory -f $(mkfile_path)/ips/pulp_cluster.mk clean
 	@make --no-print-directory -f $(mkfile_path)/ips/tbtools.mk clean
```

Comment:
is this a file generated by ipapprox?

Reply:
Yes. It's probably best then if we remove and gitignore it.